apps/settings/templates/settings/dataportability.html

```diff
@@ -2,6 +2,43 @@
 {% block settings %}
 
 <h3>Data Portability</h3>
-<p>Coming soon...</p>
+<div class="infinitive-scroll">
```

Comment:
It's not necessary to display all the bookmarks that will be exported. A simple button will suffice.

I think this can be simplified to something like:

```
<h3>Data Portability</h3>

<p>If you want to export all your bookmarks and import them to an other service, you can do so below. The downloaded
file will be a comma separated file (csv).</p>

<form action="{% url 'settings:data_portability' %}" method="post">
    {% csrf_token  %}
    <button type="submit" class="btn btn-primary">Export your bookmarks!</button>
</form>
```

There is no reason _not to_ include the `{% csrf_token %}`